pkg/resources/cloudcontroller/openstack.go

```diff
@@ -171,13 +171,19 @@ func getOSFlags(data *resources.TemplateData) []string {
 }
 
 func getOSVersion(version semver.Semver) (string, error) {
+	if version.Major() < 17 {
+		return "", fmt.Errorf("Kubernetes version %s is not supported", version.String())
```

Comment:
nit: normally you don't capitalize the first error char, this case is particular because it's a name.

Reply:
I've just copied the error message as it was before, I haven't changed anything regarding that.

Reply:
true, BTW this is not a blocker ;-)